haskell/ghc_bindist.bzl

```diff
@@ -212,8 +212,20 @@ def _ghc_bindist_impl(ctx):
 
     # On Windows the bindist already contains the built executables
     if os != "windows":
-        _execute_fail_loudly(ctx, ["./configure", "--prefix", bindist_dir.realpath])
+        _execute_fail_loudly(ctx, ["sed", "-i", "s/RelocatableBuild = NO/RelocatableBuild = YES/", "mk/config.mk.in"])
+        _execute_fail_loudly(ctx, ["./configure", "--prefix", bindist_dir.realpath, "--libdir", "${exec_prefix}/lib"])
```

Comment:
Where does `${exec_prefix}` come from?

Reply:
`${exec_prefix}` is defined within `mk/install.mk`. However, it turns out that `${exec_prefix}/lib` is the default for `RelocatableBuild = YES`. So the `--libdir` flag is redundant. I've removed it.